fastify.js

```diff
@@ -200,6 +204,16 @@ function build (options) {
   return fastify
 
   function routeHandler (req, res, params, context) {
+    if (closing === true) {
+      res.writeHead(503, {
+        'Content-Type': 'application/json',
+        'Content-Length': '80',
```

Comment:
Could you calculate it based on the payload [as with 404](https://github.com/fastify/fastify/blob/master/fastify.js#L708) ?

Reply:
That `80` is the length of the json response ;)

Reply:
I know. It's better to write `payload.length`

Reply:
**It is** the payload length.
We can do the same at the line you have pointed pointed out, since the payload will be always the same.